types/itemsjs/index.d.ts

```diff
@@ -0,0 +1,138 @@
+// Type definitions for itemsjs 2.1
+// Project: https://github.com/itemsapi/itemsjs
+// Definitions by: Adam Thompson-Sharpe <https://github.com/MysteryBlokHed>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace itemsjs;
+
+declare namespace itemsjs {
+    interface Bucket<I extends {}> {
+        key: keyof I & string;
+        doc_count: number;
+        selected: boolean;
+    }
+
+    type Buckets<I extends {}> = Array<Bucket<I>>;
+
+    interface SearchAggregation<I extends {}, A extends string> {
+        name: A;
+        title: string;
+        position: number;
+        buckets: Buckets<I>;
+    }
+
+    interface Pagination {
+        page: number;
+        per_page: number;
+        total: number;
+    }
+
+    interface SearchOptions<I extends {}, S extends string, A extends string> {
+        query?: string | undefined;
+        page?: number | undefined;
+        per_page?: number | undefined;
+        /** The name of a sort defined in the configuration's sortings, or a new custom one */
+        sort?: S | Sorting<I> | undefined;
+        filters?: Partial<Record<A, string[]>> | undefined;
+        /** A custom function to filter values */
+        filter?: ((item: I) => boolean) | undefined;
+        /** A custom function to filter values before `filter` and `filters` are considered */
+        prefilter?: ((item: I) => boolean) | undefined;
+        isExactSearch?: boolean | undefined;
+        removeStopWordFilter?: boolean | undefined;
+    }
+
+    interface AggregationOptions<A extends string> {
+        name: A;
+        page?: number | undefined;
+        per_page?: number | undefined;
+        query?: string | undefined;
+        conjunction?: boolean | undefined;
```

Comment:
Based on https://github.com/itemsapi/itemsjs/blob/master/src/lib.js#L208-L209

```suggestion
        /** @default 1 */
        page?: number | undefined;
        /** @default 10 */
        per_page?: number | undefined;
        query?: string | undefined;
        conjunction?: boolean | undefined;
```